src/mqtt/protocol/mqtt/mqtt_quic_client.c

```diff
@@ -1646,8 +1644,8 @@ quic_mqtt_pipe_start(void *arg)
 		mqtt_send_msg(NULL, s->connmsg, s);
 	}
 
-	if (!nni_aio_list_active(&s->time_aio))
-		nni_sleep_aio(s->retry * NNI_SECOND, &s->time_aio);
+	// if (!nni_aio_list_active(&s->time_aio))
+	//	nni_sleep_aio(s->retry * NNI_SECOND, &s->time_aio);
```

Comment:
These two lines should be moved to somewhere instead of commenting.
@RanMaoyi Plz move to somewhere better